src/tests/routes/controllers/users.test.js

```diff
@@ -190,4 +190,17 @@ describe('User stats', () => {
       expect(body.data.count).to.be.eql(2);
     });
   });
+
+  context('User Reports', () => {
+    it('should return a 200 and get comments by a user', async () => {
+      const { status, body } = await chai
+        .request(app)
+        .get('/api/v1/users/comments')
+        .set('x-access-token', userToken);
+
+      expect(status).to.eql(200);
+      expect(body).to.have.keys('status', 'message', 'data');
+      expect(body.message).to.eql('Successfully retrieved all comments');
```

Comment:
Let's test for the value of the response data.

Reply:
this has been resolved too